jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -200,7 +201,7 @@ def test_filelist(self, imviz_helper, tmp_path):
 
         # Generate some files to parse.
         for i in range(2):
-            fpath = tmp_path / f'myfits_{i}.fits'
+            fpath = tmp_path / imviz_helper.app.return_data_label('myfits.fits')
```

Comment:
This broke the test. I am generating filenames here, not data label. There is no need to change anything here. The point of the test is to pass in a list of filenames.